lib/web/databases.go

```diff
@@ -617,7 +627,7 @@ func (s *databaseInteractiveSession) Run() error {
 
 func (s *databaseInteractiveSession) Close() error {
 	s.replConn.Close()
-	return s.ws.Close()
```

Comment:
Where is the websocket getting closed now?

Reply:
The middleware always closes it.

https://github.com/gravitational/teleport/blob/e356649f3c61f8d94c6f9b59e1858d813eaef157/lib/web/apiserver.go#L4456-L4462